### PR TITLE
revert: remove robots, diffuser, drone, and custom auth

### PR DESCRIPTION
**Pull request by novice0192**
revert: remove diffuser, drone, and otp based user verification

revert: remove diffuser, drone, and otp based user verification

revert: remove population :(

### CODE REVIEW

File: src/api/attack/util.rs [hunk 1]
```diff
@@ -793,88 +653,3 @@ pub fn get_attacker_types(conn: &mut PgConnection) -> Result<HashMap<i32, Attack
         })
         .collect::<HashMap<i32, AttackerType>>())
 }
-
-pub fn get_buildings(
```
Comment (DonWick32) [inline src/api/attack/util.rs:797]: Why is this removed?

File: src/api/auth/util.rs [hunk 2]
```diff
@@ -1,34 +1,12 @@
...
 use crate::util::function;
 use crate::{constants::INITIAL_RATING, error::DieselError};
 use anyhow::Result;
 use diesel::prelude::*;
-use pwhash::bcrypt;
 use rand::{distributions::Alphanumeric, Rng};
 use redis::Commands;
-use std::time::{SystemTime, UNIX_EPOCH};
-
-pub fn get_user(conn: &mut PgConnection, id: i32) -> Result<Option<User>> {
```
Comment (DonWick32) [inline src/api/auth/util.rs:13]: This need not be removed rt?

File: src/api/attack/util.rs (hunk 1 above)
Comment (novice0192) [reply src/api/attack/util.rs:797]: This was unused after removing diffuser and drone

File: src/api/auth/util.rs (hunk 2 above)
Comment (novice0192) [reply src/api/auth/util.rs:13]: This was unused after removing otp based password reset and verification
